package.json

```diff
@@ -314,6 +314,12 @@
         "mac": "cmd+r",
         "when": "editorLangId == sql && resourceExtname != .inb"
       },
+      {
+        "command": "vscode-db2i.runEditorStatement.inView",
+        "key": "ctrl+alt+r",
+        "mac": "cmd+ctrl+r",
+        "when": "editorLangId == sql"
```

Comment:
Should this have the same resource extension check as at line 315?

Reply:
I didn't think so. It might be nice to take a statement from a Notebook and run it in a new Window